src/main/json2xlsx.js

```diff
@@ -55,4 +48,20 @@ const run = async (args) => {
   console.log('done.');
 };
 
+async function readSheetData (fileMap, file, sheetDirectory) {
```

Comment:
There is absolutely no point passing both `fileMap` and `file`. Function should only know about files that it should read not about any other files. It reminds me slightly passing whole application context only so that function logic reaches out for one tiny part of that it.